### DISCUSSION TITLE
How to modify upscale of BasicVSR or  RealBasicVSR

### DISCUSSION THREAD

**Discussion by mayunjie** [open-mmlab/mmediting#1196] [How-to] [answered]
i want to modify upscale to x2 or x3，how to set

**Comment by wangruohui** [comment, marked as answer]
Currently, BasicVSR is hard-coded to be 4x. There are two upsampling modules in BasicVSRNet. If you want to change it to 2x you need to remove one of them. Moreover, you need to modify the data pipeline to supply 2x data for computing ground-truth.
